metricbeat/module/kibana/status/data.go

```diff
@@ -0,0 +1,43 @@
+package status
+
+import (
+	"encoding/json"
+
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/metricbeat/schema"
+	c "github.com/elastic/beats/metricbeat/schema/mapstriface"
+)
+
+var (
+	schema = s.Schema{
+		"uuid": c.Str("uuid"),
+		"name": c.Str("name"),
+		"version": c.Dict("version", s.Schema{
+			"number": c.Str("number"),
+		}),
+		"status": c.Dict("status", s.Schema{
```

Comment:
@tsg I wanted to do here `state.overall = status.overal.state` but couldn't find a way in schema. Any hints?